catalog/schematron/catalog-schematron-plugin/pom.xml

```diff
@@ -49,6 +49,12 @@
             <groupId>ddf.platform.util</groupId>
             <artifactId>platform-util</artifactId>
         </dependency>
+        <dependency>
```

Comment:
I needed bumped the version up for junit so that I could use the @ClassRule annotation, which let's me make the temporary folder and setup static. Should I just use @Rule and @Before? I also just realized I didn't do that consistently across the two unit test classes I made.